app/gui2/src/components/visualizations/TableVisualization.vue

```diff
@@ -131,6 +131,8 @@ const SQLITE_CONNECTIONS_NODE_TYPE =
   'Standard.Database.Internal.SQLite.SQLite_Connection.SQLite_Connection'
 const POSTGRES_CONNECTIONS_NODE_TYPE =
   'Standard.Database.Internal.Postgres.Postgres_Connection.Postgres_Connection'
+const SNOWFLAKE_CONNECTIONS_NODE_TYPE =
```

Comment:
Removing `private` from implementation classes isn't long term solution. What's the envisioned long term solution?

Reply:
It turns out that `Snowflake_Connection` is not an implementation class. It is a public facing type that cannot be private because it is being used by users.

If we want to hide it, we'd probably need a way to have a generic `Connection` interface that is public and `Snowflake_Connection` would only implement it.

**However**, it is not that simple, as we do provide some 'additions' - e.g. `Snowflake_Connection` has additional `warehouse` method that is Snowflake-specific - no other DBs have it. So I think it is actually a public interface.

Though that suggests I should probably move it out of `Internal`.

Reply:
OK. If it is needed by end users, than it shouldn't be in `Internal`.

Reply:
Indeed, I've moved it.